setuptools/command/build_py.py

```diff
@@ -256,6 +256,7 @@ class _IncludePackageDataAbuse:
     """Inform users that package or module is included as 'data file'"""
 
     MESSAGE = """\
+    Installing {importable!r} as data is deprecated, please list it as `package`.
```

Comment:
```suggestion
    Installing {importable!r} as data is deprecated, please list it in `packages`.
```

Let's use the actual key, or not use the backticks, as this could be confusing.